### PR TITLE
feat :text list & nav done

### PR DESCRIPTION
**Pull request by Developers-codz**
Added Text utils, list and nav page

### CODE REVIEW

**Review by vrx29 — APPROVED** [review]
Looks good to me.

File: css/style/navigation.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    display:grid;
+    justify-content: space-between;
+    grid-template-columns: repeat(2,1fr);
+}
+.dropdown{
+    position: relative;
+    margin:0rem 1rem;
+}
+.dropbtn{
+    background-color: var();
```
Comment (arjun-computer-geek) [inline css/style/navigation.css:16]: why you use css variable if there is no value in it?

Comment (arjun-computer-geek) [inline css/style/navigation.css]: hey, Why are you using css variable if there is no value in it?

**Review by arjun-computer-geek — CHANGES_REQUESTED** [review]
Hey do some change what i mentioned.

File: css/style/navigation.css (hunk 1 above)
Comment (Developers-codz) [reply css/style/navigation.css]: That's by mistake, corrected it

**Review by anik31 — APPROVED** [review]
LGTM

**Pull request merged**